Search/SearchService.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SearchService.swift
+//  Search
+//
+//  Created by David Haynes on 10/03/2016.
+//  Copyright © 2016 Ordnance Survey. All rights reserved.
+//
+
+import Fetch
+
+/// Service for consuming the OS Places API
+public class SearchService: Searchable {
```

Comment:
Wonder whether the name of this class should be more specific about searching OS Places? To allow for some of the other possible sources of data as the Android framework is including.